tests/examples/opinions/united_states/tenn_example.compare.json

```diff
@@ -1,351 +1,351 @@
+    "docket_numbers": "M2022-00471-SC-R11-CV",
+    "judges": "Chief Justice Holly Kirby",
+    "lower_courts": "Coffee",
+    "summaries": "In this appeal regarding Tennessee's survival statute, Tennessee Code Annotated section 20-5-102, we hold that a cause of action for invasion of privacy based on intrusion upon seclusion does not abate upon the death of the person whose privacy was invaded. Here, an elderly woman with severely impaired cognitive functioning was a resident of the defendant skilled nursing facility. While an employee of the facility was helping the resident take a shower, the employee took a personal video call with her incarcerated boyfriend. During the call, the employee propped her cell phone in the shower in a way that displayed the resident's nude body to the boyfriend. The resident's conservator sued the defendant facility claiming, inter alia, invasion of the resident's privacy. The trial court granted summary judgment in favor of the facility, and the plaintiff appealed. While the appeal was pending, the resident died, and her estate was substituted. The Court of Appeals reversed; it held the complaint stated a claim for invasion of privacy based on intrusion upon seclusion, and that the claim did not abate upon the death of the resident. We granted the defendant facility permission to appeal on abatement. Tennessee Code Annotated section 20-5-102 provides that no civil action based on \"wrongs\" abates upon the death of either party, \"except actions for wrongs affecting the character of the plaintiff.\" On appeal, the defendant facility argues that the claim abated either because section 20-5-102 does not apply at all, or because the character exception in the statute applies, since a claim for intrusion upon seclusion is an action for \"wrongs affecting the character of the plaintiff.\" We hold that the survival statute, section 20-5-102, applies to the claim of intrusion upon seclusion, and that the exception for actions for \"wrongs affecting the character of the plaintiff\" does not apply. Therefore, we affirm the holding of the Court of Appeals that the claim for intrusion upon seclusion did not abate upon the death of the resident whose privacy was invaded.",
     "case_name_shorts": "",
     "per_curiam": false
   },
   {
-    "case_dates": "2023-09-28",
-    "case_names": "Commercial Painting Company, Inc. v. The Weitz Company, LLC",
-    "download_urls": "/sites/default/files/OpinionsPDFVersion/COMMERCIAL-Majority%20Opinion.pdf",
+    "case_dates": "2025-03-26",
+    "case_names": "Heather Smith v. BlueCross BlueShield of Tennessee (Concurring)",
```

Comment:
Looks like this contains separate concurrences and dissents - we should eventually combine this with the other opinions, but perhaps we should include the type, here concurrence and remove it from the case name.  I think this is still a summary which is nice

Reply:
thankfully it does appear to be in the URL as well 

`/default/files/OpinionsPDFVersion/Separate%20Opinion%20`

vs. 

`les/OpinionsPDFVersion/Majority%20Opinion%20-%2`